src/snagflash/dfu.py

```diff
@@ -34,7 +34,7 @@ def dfu_cli(args):
 	dev.default_timeout = int(args.timeout)
 	for dfu_config in args.dfu_config:
 		(altsetting,sep,path) = dfu_config.partition(":")
-		if args.size:
```

Comment:
This is actually a deprecated command line arg which is why you got an AttributeError. Could you please remove this check entirely?

Reply:
removed